Moodboard-program/moodboard-prototype

```diff
@@ -0,0 +1,262 @@
+import sys
+import time
+
+
+
+def paintblack(canvas2):
+  pixels = getPixels(canvas2)
+  for i in pixels:
+    setColor(i,black)
+    
+
+def start():
+  #load image from file
+  canvas2=makePicture(r".\Ship-image-structure\ship+biodomes.jpg")
+  show(canvas2)
+  #get input from user
+  
+  keypress = requestNumber("Press 1 for biodome or 2 for ship: ")
+  #if the user entered 1 then zoom into the top section of the image
+  #if the user enters 2(or anything else) zoom into the bottom part of the image
+  
+  if keypress == 1:
+    paintblack(canvas2)
+    repaint(canvas2)
+    EnlargeImageTopSection()
+    time.sleep(2)
+  elif keypress == 2:
+    paintblack(canvas2)
+    repaint(canvas2)
+    EnlargeImageBottomSection()
+    time.sleep(2)
+
+
+def EnlargeImageTopSection():
+  #get the image and then make an empty image 
+  file=(r".\Ship-image-structure\ship+biodomes.jpg")
+  src=makePicture(file)
+  
+  canvas = makeEmptyPicture(1326,698)
+  canvas2 = makeEmptyPicture(1326,698)
+  
+  
+  #Display the image in the empty image
+  canvas = src
+  
+  #get pixels at source X
+  #paste horizontal pixels from pixel 100 to pixel 2000
+  #get pixels at source Y
+  #paste vertical pixels from pixel 100 to pixel 1000
+  #scale up by 60%
+ 
+  sourceX = 391
+  for targetX in range (10,1000):
+    sourceY = 146
+    for targetY in range(100,500):
+      srcpx = getPixel(src,int(sourceX),int(sourceY))
+      color = getColor(srcpx)
+      setColor(getPixel(canvas2,targetX,targetY), color)
+      sourceY = sourceY + 0.6
+    sourceX = sourceX + 0.6
+  repaint(canvas2)
+  
+  
+  paintblack(canvas2)
+  repaint(canvas2)
+  
+  #load the biodome image and display it on the canvas
+  pic2=makePicture(r".\Ship-image-structure\biodome.jpg")
+  for pixel in pic2.getPixels():
+    x = getX(pixel)
+    y = getY(pixel)
+    color = getColor(pixel)
+    target = getPixelAt(canvas2, x, y) 
+    setColor(target, color)
+  repaint(canvas2)
+  
+
+    
+  
+  
+  #get user number input and store in variable
+  #if the variable is equal to 1 paint the canvas black then load image from file and paint to canvas
+  #if the variable is equal to 2 paint the canvas black then load image from file and paint to canvas
+  #if the variable is equal to 3 paint the canvas black then load image from file and paint to canvas
+  #if the variable is equal to 4 paint the canvas black then load image from file and paint to canvas
+
```

Comment:
Again, these comments will be more effective placed above the relevant sections of code